HornsAndHooves-flat_map.gemspec

```diff
@@ -19,8 +19,8 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   # specify any dependencies here; for example:
-  s.add_dependency(%q<activesupport>, ["> 4.0", "< 4.2"])
-  s.add_dependency(%q<activerecord>, ["> 4.0", "< 4.2"])
+  s.add_dependency(%q<activesupport>, ["> 4.0", "< 5"])
```

Comment:
With Rails 5 coming out, there won't be a Rails 4.3. And this still supports Rails 4.1.